lib/chart/line/view.ex

```diff
@@ -0,0 +1,58 @@
+defmodule Chart.Line.View do
+  @moduledoc false
+
+  alias Chart.Internal.Utils
+
+  @gap_from_x_axis_line 30
+  @gap_from_y_axis_line 35
+
+  def get_axis(settings, vector) do
+    Utils.find_axis_for_vector(settings, vector) |> hd()
```

Comment:
Existuje fce `Enum.filter/2`.

Reply:
Proc nepostacuje ten `for`?

Reply:
Filtrujes nebo cyklujes? For se moc nepozuiva, tedy pouziva se, pokud to dava smysl, napr.

```elixir
for x <- 1..10, y <- 10..20, z <- 20..30 do
  x * y * z
end
```